downscale/estimate_forcing_regression.f90

```diff
@@ -81,6 +82,32 @@ subroutine read_station_weights(sta_weight_name, & !input
       integer(I4B), intent(inout):: error
     end subroutine read_station_weights
 
+
+    subroutine read_nwp(currentTime,nwp_timestep_file,nPredict,n_nwp,nwp_vars,station_grid,x,z,error)
```

Comment:
let's make this more general, eg 'read_,dynamic_predictor_grid' ... as it might be sat, rean. etc.